tox.ini

```diff
@@ -122,12 +121,23 @@ commands =
     {[testenv:yamllint]commands}
 
 [testenv:doc]
+description = Invoke sphinx-build to build the HTML docs
 # doc requires py3 due to use of f'' strings and using only python3 as
 # basepython risks using python3.4 which is not supported.
 basepython = python3.7
 passenv = *
+usedevelop = False
 commands =
-    python setup.py build_sphinx -n -W --builder=html
+    python -m sphinx \
+      -a -n -W \
+      -b html --color \
+      -d "{toxworkdir}/docs_doctree" \
+      docs/ "{toxworkdir}/docs_out"
+
+    # Print out the output docs dir and a way to serve html:
+    python -c \
+    'import pathlib; '\
+    'docs_dir = pathlib.Path(r"{toxworkdir}") / "docs_out"; index_file = docs_dir / "index.html"; print(f"\nDocumentation available under `file://\{index_file\}`\n\nTo serve docs, use `python3 -m http.server --directory \{docs_dir\} 0`\n")'
```

Comment:
I would rather see this in the documentation instead than here for maintenance purposes.

Reply:
Perhaps this would be a good candidate for a more readable `./bin/output-docs-path` script? We have some other stuff in this file (like the setuptools_scm version generation) that might be more easy to maintain if it lived in a script directory. Maybe for another change ...

Reply:
I have to think about this. Ideally, I'd not want to maintain this away from the context which is concentrated right here. I with `tox` had better support for multiline commands...
Currently, I'd want to avoid introducing yet another directory...